### ISSUE TITLE
Rename zksync to zkSync - https://github.com/wevm/viem/blob/main/src/zksync/utils/getEip712Domain.ts#L24C14-L24C20da

### ISSUE THREAD

**Issue by zpoken** [wevm/viem#2607]
### Check existing issues

- [X] I checked there isn't [already an issue](https://github.com/wevm/viem/issues) for the bug I encountered.

### Viem Version

latest

**Comment by zpoken** [comment]
see there -https://github.com/zksync-sdk/zksync-ethers/blob/main/src/signer.ts#L83

**Comment by tmm** [comment]
This was changed specifically by @JackHamer09 as part of https://github.com/wevm/viem/pull/2546

**Comment by Vadim121197** [comment]
@tmm will this be added in the next version?

**Comment by tmm** [comment]
Been out since 2.19

https://github.com/wevm/viem/releases/tag/viem%402.19.0

**Comment by Vadim121197** [comment]
![image](https://github.com/user-attachments/assets/79a334c2-53c3-4d80-931c-1e155e419118)

**Comment by Vadim121197** [comment]
![image](https://github.com/user-attachments/assets/19dc6df8-a3e0-49fa-a48c-c7a418270dc1)

**Comment by Vadim121197** [comment]
Please, check again(

**Comment by Vadim121197** [comment]
@tmm

**Comment by jxom** [comment]
![image](https://github.com/user-attachments/assets/e32574c9-e000-46aa-9be8-af5a8b3b6994)

**Comment by zpoken** [comment]
viem v2.19.3(latest version), function [getEip712Domain](https://github.com/wevm/viem/blob/main/src/zksync/utils/getEip712Domain.ts#L12)
```ts
domain: {
      name: 'zksync', // TODO replace zksync => zkSync
},
```
from there - https://github.com/wevm/viem/blob/main/src/zksync/utils/getEip712Domain.ts#L24

**Comment by zpoken** [comment]
did you close the issue without checking it?

**Comment by jxom** [comment]
Fixed in 2.19.4
